### PR TITLE
Update README with ucm env var instructions

### PR DESCRIPTION
**Pull request by hojberg**
## Overview
Simple and quick update to the README to include the new configuration options from https://github.com/unisonweb/unison/pull/1845

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +37,14 @@ $ stack build && stack exec tests && stack exec unison
...
+Codebase Server
+---------------
+
+When `ucm` starts it starts a Codebase web server that is used by the 
+[Codebase UI](https://github.com/unisonweb/codebase-ui). It selects a random
+port and a unique token that most be used when starting the UI to correctly
+connect to the server.
+
+The port, host and token can all be configured by providing environment
+variables when starting `ucm`: `UCM_PORT`, `UCM_HOST`, and `UCM_TOKEN`.
```
Comment (hojberg) [inline README.md:50]: @stew from https://github.com/unisonweb/unison/pull/1845 I wasn't sure if its `UCM_HOST` or `UCM_BIND`?

**Review by pchiusano — APPROVED** [review]
LGTM but stew should confirm the variable names. Looks like an empty file accidentally got checked in.

**Comment by aryairani** [comment]
Yeah it might be a little tricky once it's already checked in, but it's a good idea to add `.DS_Store` and other OS-specific stuff to your global git ignore settings.

**Comment by hojberg** [comment]
@aryairani definitely — I always forget to add it when I reinstall my machine :D

**Comment by stew** [comment]
I named the variable UCM_BIND i'm open to either name

**Comment by pchiusano** [comment]
@hojberg cool. You can merge after fixing the variable name to match UCM_BIND as @stew said or I guess switch the name to UCM_HOST in the code if you have strong feelings about it (it sounds like no one else does).

**Comment by hojberg** [comment]
Went with `UCM_HOST` - it feels a little more intuitive to me.

**Pull request merged**